### DISCUSSION TITLE
Servo outputs with Jeti Receivers

### DISCUSSION THREAD

**Discussion by ghost** [iNavFlight/inav-configurator#1698] [Q&A]
Hello,
having big troubles to setup the FC405 WMN with my Jeti DC16.
Within  INAV under chapter "Receiver", all servos "throttle, Roll, Yaw and Pitch are visible and working

Under chapter "outputs" I get only minimal amplitudes for roll yaw and pitch
throttle will stay at 0 for the motor.
Throttle could be activated at chapter "outputs" via SW switch at the INAV SW screen
But no activation at FC405 WMN
Any help would be appreciated
Best regards

**Comment by MrD-RC** [comment]
There is nothing wrong with the receiver connection if you are seeing the bars move on the **Receiver** page.

If your servos are not moving in the bars on the **outputs** page, have you set up a mixer in the **mixer** page? Have you enabled motor and servo outputs?

**Reply by ghost** [reply]
Thank you Daren. Receiver is working perfectly even with JETIEXBUS connection.
Mixer was set automatically. But maybe to "enable motor and servo outputs" if this is a separated step?
Where do I find this? 
Thank you for your help
Best regards
Markus

**Reply by ghost** [reply]
Hello, so everything is running now: BESIDES the Motor. I dont get the Motor to run if activated by RC.
Motor activated by INAV SW is ok

I checked now everything once again, set the "enable motor and servo output" but absolutely no success at motor output.
At receiver, everything is shown correctly. Throttle is in the range from 1000 to 2000 working correctly. 
But no reaction at the motor itself.
Is there any other switch / setup ect which could prevent motor from working?
Regards
Markus

**Reply by MrD-RC** [reply]
It is probably because of GPS fix or nav mode enabled. Check the CLI with the **status** command or the OSD system messages in your goggles. 

Please stop creating issues. The problems you're having are all setup. Not problems with the firmware. 

What did you change to stop the receiver saying it was in failsafe? The information may be useful for others.

**Comment by ghost** [comment]
HI Daren, thank you for supporting me
Sorry if I created some trouble, this is not my intention.
I thought this forum could be used for interested people if they have trouble with INAV?

After CLI status command, the system tells me:
Arming disabled flags: HWFAIL CLI
Maybe you could give me a hint, what this means, because the HW runs now. 
Slightly problems getting 6 Sats, but strange for me is, that if FC had up to 10 sats, if than only 1 is lost (means 9 sats completely then) the GPS Symbol at  the top of INAV switches from blue to red.
And with red GPS symbol, arming is not possible.
In general, is it the normal procedure that you have to switch 5 times into Armed Mode to reach Armed Mode?
Is there no other procedure? Normally I would expect, that activating the arming mode should be easy, by switching to Armed Mode  with only one  activation

**Reply by MrD-RC** [reply]
No problem. This isn't really a forum like that. It is more for reporting problems with INAV. But setup queries are answered too. A Better place for setup would be Discord. But keep replying to this discussion with your problems.

Any arming disabled flags can be looked up at https://github.com/iNavFlight/inav/wiki/%22Something%22-is-disabled----Reasons

`HWFAIL` shows that a device is not working. Look on the top bar of Configurator for devices in red, or go to the Setup page and look at the list on the right.

For arming, it isn't just seeing 6 satellites. There must be 3D fixes with at least 6 satellites. This is why you may have 7 satellites, but still can't arm in some cases.

**Reply by MrD-RC** [reply]
You should not arm normally by rapidly switching the arm switch. That is a bypass for testing only. It bypasses the safety checks.

**Comment by ghost** [comment]
one more question, which might be really a malfunction of inav
arming is now possible all time, but motor is only to be activated if ROLL is done one time before !
Is this really the case, is this normal behaviour?
I can only acitivate motor after ROLL is done one time beforehand
Please give me short feedback
Thank you
Best regards
Markus
